ansible/roles/nginx-prometheus/templates/nginx.conf

```diff
@@ -8,7 +8,17 @@ events {
   worker_connections 448;
 }
 
+lock_file /run/ngxprom.lock;
```

Comment:
Ah, lol, I figured it out from another PR: it's for `prometeus`. (Maybe a comment would help.)